cmd/logger/target/http/http.go

```diff
@@ -66,11 +66,13 @@ func (h *Target) startHTTPLogger() {
 
 			resp, err := h.client.Do(req)
 			if err != nil {
+				h.client.CloseIdleConnections()
 				continue
 			}
 
 			// Drain any response.
 			xhttp.DrainBody(resp.Body)
+			h.client.CloseIdleConnections()
```

Comment:
Why do we want to do this. This will effectively disable any chance of keepalive AFAICT.

(same for the other ones)

Reply:
I don't think it disables keep-alives. Only idle connections are closed.

Reply:
Oh yes you are right I need to do this only during error.